src/components/v1/V1Create/index.tsx

```diff
@@ -333,10 +336,11 @@ export default function V1Create() {
                 color: viewed
                   ? colors.text.primary
                   : colors.text.action.primary,
-                border: viewed
-                  ? '1px solid ' + colors.stroke.secondary
-                  : '1px solid' + colors.stroke.action.primary,
-                borderLeftWidth: active ? 10 : 1,
+                borderColor: viewed
```

Comment:
Passing fix. This fixes a console error where apparently React doesn't like it when styles are overridden. In this case `borderLeftWidth` was overriding the above `border` style.